ktor-server/ktor-server-plugins/ktor-server-cors/common/src/io/ktor/server/plugins/cors/routing/CORS.kt

```diff
@@ -23,6 +23,15 @@ import io.ktor.server.plugins.cors.*
  *
  * [Report a problem](https://ktor.io/feedback/?fqname=io.ktor.server.plugins.cors.routing.CORS)
  */
+
+@Deprecated(
+    message = "The plugin installation has been moved to the Route.cors method",
```

Comment:
let's explain the difference between `install(CORS)` and `cors {}` in the API docs for the both symbols